pom.xml

```diff
@@ -135,11 +144,9 @@
                                     <!--skip>true</skip-->
                                     <shapes>
                                         <include>target/dist/validation/COLLECTION_QUDT_QA_TESTS_ALL.ttl</include>
-                                        <include>target/dist/schema/shacl/SCHEMA_QUDT_NoOWL.ttl</include>
```

Comment:
We lost two lines in pom.xml by copying the pom.xml from the main branch - the lines that actually cause the cardinalities to be checked. 

Rebase would have been the way to go